### DISCUSSION TITLE
General Help

### DISCUSSION THREAD

**Discussion by RobyRew** [SpotCompiled/SpotilifeC#1] [General]
Using Pocket App ([Downloaded from the AppStore](https://apps.apple.com/es/app/postbox-pocket/id1644577838?l=en)) I added julioverne's repo there and I press on download .deb file, and it redirects me to Safari and I copied the file link:
https://julio.hackyouriphone.org/debfiles/com.julioverne.spotilife_1.8_iphoneos-arm.deb

So here you have, whatever deb file you want to download use Pocket and you would have direct download for the deb file.

**Comment by yodaluca23** [comment]
> Using Pocket App ([Downloaded from the AppStore](https://apps.apple.com/es/app/postbox-pocket/id1644577838?l=en)) I added julioverne's repo there and I press on download .deb file, and it redirects me to Safari and I copied the file link:
> https://julio.hackyouriphone.org/debfiles/com.julioverne.spotilife_1.8_iphoneos-arm.deb
> 
> So here you have, whatever deb file you want to download use Pocket and you would have direct download for the deb file.

![image](https://user-images.githubusercontent.com/67206487/200077195-f8f39ad0-bf60-4904-80fc-281c35bee086.png)

Rip, it's fine I use [this](https://github.com/borishonman/cydownload/releases) and it works fine right now...

**Comment by yodaluca23** [comment]
@RobyRew  I tried JailBreaking using unc0ver but... I found my old phone...

saw it was on 12.5.5 and saw that Unc0ver supported 11-14 so I figured I'd update to 12.5.6, because why not? Then I realize that's the ONE VERSION not supported..... IF I HAD JUST DECIDED NOT TO UPDATE, I WOULD BE ABLE TO JAILBREAK I AM SO MAD!!!!

**Reply by RobyRew** [reply]
> @RobyRew  I tried JailBreaking using unc0ver but... I found my old phone...
> 
> 
> 
> saw it was on 12.5.5 and saw that Unc0ver supported 11-14 so I figured I'd update to 12.5.6, because why not? Then I realize that's the ONE VERSION not supported..... IF I HAD JUST DECIDED NOT TO UPDATE, I WOULD BE ABLE TO JAILBREAK I AM SO MAD!!!!
> 
> 
> 
> 

Don't worry, Checkra1n IS HERE TO SAVE YOU FOREVER!!!!!
download from their only official repo:
https://checkra.in/

Remember to: connect the phone using a good lightning cable, and on Checkra1n's options to allow non supported versions, and verbose, also activating the "Quick something"
then hit Start and boom you need to follow the instructions on Checkra1n's app wen appear, any problem send message to me I'll try to help ASAP. ;)

**Reply by yodaluca23** [reply]
> > @RobyRew  I tried JailBreaking using unc0ver but... I found my old phone...
> > 
> > 
> > 
> > saw it was on 12.5.5 and saw that Unc0ver supported 11-14 so I figured I'd update to 12.5.6, because why not? Then I realize that's the ONE VERSION not supported..... IF I HAD JUST DECIDED NOT TO UPDATE, I WOULD BE ABLE TO JAILBREAK I AM SO MAD!!!!
> > 
> > 
> > 
> > 
> 
> Don't worry, Checkra1n IS HERE TO SAVE YOU FOREVER!!!!!
> download from their only official repo:
> https://checkra.in/
> 
> Remember to: connect the phone using a good lightning cable, and on Checkra1n's options to allow non supported versions, and verbose, also activating the "Quick something"
> then hit Start and boom you need to follow the instructions on Checkra1n's app wen appear, any problem send message to me I'll try to help ASAP. ;)

Is the website down? It's just timing me out... and I was doing some research in Checkra1n, does it have a windows version? And does it keep the applications sandboxed like unc0ver?

Also, is it as well-known as unc0ver cause, I've never heard of it.

**Reply by RobyRew** [reply]
> Is the website down? It's just timing me out... and I was doing some research in Checkra1n, does it have a windows version? And does it keep the applications sandboxed like unc0ver?
> 
> Also, is it as well-known as unc0ver cause, I've never heard of it.

It's 100% legit. Checkra1n is a semi-tethered jailbreak developed by some of the best Jailbreak devs on the community.
Here's a copy of Checkra1n from my Mac.
[checkra1n_v0.12.4.zip](https://github.com/SpotCompiled/SpotilifeC/files/9983262/checkra1n_v0.12.4.zip)
If you're using Windows or Linux you can use a bootable usb to boot into checknix and jailbreak from there. https://github.com/asineth0/checkn1x
Download de ISO, use [Rufus](https://github.com/asineth0/checkn1x) to make the bootable and follow de instructions.

**Reply by yodaluca23** [reply]
> > Is the website down? It's just timing me out... and I was doing some research in Checkra1n, does it have a windows version? And does it keep the applications sandboxed like unc0ver?
> > Also, is it as well-known as unc0ver cause, I've never heard of it.
> 
> It's 100% legit. Checkra1n is a semi-tethered jailbreak developed by some of the best Jailbreak devs on the community. Here's a copy of Checkra1n from my Mac. [checkra1n_v0.12.4.zip](https://github.com/SpotCompiled/SpotilifeC/files/9983262/checkra1n_v0.12.4.zip) If you're using Windows or Linux you can use a bootable usb to boot into checknix and jailbreak from there. https://github.com/asineth0/checkn1x Download de ISO, use [Rufus](https://github.com/asineth0/checkn1x) to make the bootable and follow de instructions.

Could I just put that ISO in a VM???

**Reply by yodaluca23** [reply]
> > Is the website down? It's just timing me out... and I was doing some research in Checkra1n, does it have a windows version? And does it keep the applications sandboxed like unc0ver?
> > Also, is it as well-known as unc0ver cause, I've never heard of it.
> 
> It's 100% legit. Checkra1n is a semi-tethered jailbreak developed by some of the best Jailbreak devs on the community. Here's a copy of Checkra1n from my Mac. [checkra1n_v0.12.4.zip](https://github.com/SpotCompiled/SpotilifeC/files/9983262/checkra1n_v0.12.4.zip) If you're using Windows or Linux you can use a bootable usb to boot into checknix and jailbreak from there. https://github.com/asineth0/checkn1x Download de ISO, use [Rufus](https://github.com/asineth0/checkn1x) to make the bootable and follow de instructions.

What settings do I check for Rufus?
![image](https://user-images.githubusercontent.com/67206487/201220206-d268e597-f908-4e50-9244-5a92884cb8a3.png)

Also, Frida-Dump looks really complicated, especially for Windows, is there any app that can do some of that automatically? That's open source and/or known?

**Reply by yodaluca23** [reply]
Also, @RobyRew  Have you heard of [Scarlet? ](https://usescarlet.com/)I just found out about it, and it looks pretty cool.

It has a ton of ads on the site though...

**Reply by RobyRew** [reply]
> Could I just put that ISO in a VM???

No, it needs to run on bare metal.

---

> What settings do I check for Rufus?
> <img src="https://user-images.githubusercontent.com/67206487/201220206-d268e597-f908-4e50-9244-5a92884cb8a3.png" alt="drawing" width="220" height="300"/>

Partition scheme I recommend GPT,  File System select exFAT and uncheck "Extended label and icon files" and "Check device for bad blocks"

---

> Also, Frida-Dump looks really complicated, especially for Windows, is there any app that can do some of that automatically? That's open source and/or known?
Yes, there's a method using the iPhone Jailbreak

https://www.ios-repo-updates.com/repository/diatrus-tvos-essentials/package/com.level3tjg.bfdecrypt/

On cydia and add the repo, then install the tweak.
After doing that go to settings search bfdecrypt and enable the toggle, then run that toggled app.
You would have a decrypted IPA on a folder don't remember exactly where.

On _NewTerm2's command line tool_ search the filesystem for the IPA like so:
`find /var/mobile/Containers/Data/Application/ -name decrypted-app.ipa`

---

> Also, @RobyRew Have you heard of [Scarlet? ](https://usescarlet.com/)I just found out about it, and it looks pretty cool.
It has a ton of ads on the site though...

Never heard about it, and it looks very sketchy I don't recommend you even click to the link's website.
That kind of web pages are normally adware hubs, just trash.

**Reply by yodaluca23** [reply]
> > Also, Frida-Dump looks really complicated, especially for Windows, is there any app that can do some of that automatically? That's open source and/or known?
>  Yes, there's a method using the iPhone Jailbreak
> 
> https://www.ios-repo-updates.com/repository/diatrus-tvos-essentials/package/com.level3tjg.bfdecrypt/
> 
> On cydia and add the repo, then install the tweak. After doing that go to settings search bfdecrypt and enable the toggle, then run that toggled app. You would have a decrypted IPA on a folder don't remember exactly where.
> 
> On _NewTerm2's command line tool_ search the filesystem for the IPA like so: `find /var/mobile/Containers/Data/Application/ -name decrypted-app.ipa`

That's great! So after I jailbreak, install either AppStore++ or AppAdmin and install Spotify Version: 8.7.46. Then install bfdecrypt and turn it on for Spotify, run Spotify, then I'll have the Dycrypted IPA somewhere in the files?

Also, is [this](https://repo.dynastic.co/package/com.spos) the official Sposify repo?

**Reply by RobyRew** [reply]
> > On _NewTerm2's command line tool_ search the filesystem for the IPA like so: `find /var/mobile/Containers/Data/Application/ -name decrypted-app.ipa`
> 
> 
> 
> That's great! So after I jailbreak, install either AppStore++ or AppAdmin and install Spotify Version: 8.7.46. Then install bfdecrypt and turn it on for Spotify, run Spotify, then I'll have the Dycrypted IPA somewhere in the files?
> 
>
> 
> Also, is [this](https://repo.dynastic.co/package/com.spos) the official Sposify repo?

Yes, if you're using iOS12 use AppStore++ from cokepokes repo it should work well.

An yes, the link you provided for Sposify is the official one

**Reply by yodaluca23** [reply]
> Partition scheme I recommend GPT, File System select exFAT and uncheck "Extended label and icon files" and "Check device for bad blocks"

The only two options for File Systems are FAT32 and NTFS...
![image](https://user-images.githubusercontent.com/67206487/201447661-051278c1-87bf-4516-b040-fa1670f17a5b.png)

And, also the repo you gave for bfdycrypt looks to be for TVOS... is that right?

**Reply by RobyRew** [reply]
> > Partition scheme I recommend GPT, File System select exFAT and uncheck "Extended label and icon files" and "Check device for bad blocks"
> 
> 
> 
> The only two options for File Systems are FAT32 and NTFS...
> 
> ![image](https://user-images.githubusercontent.com/67206487/201447661-051278c1-87bf-4516-b040-fa1670f17a5b.png)
> 

Then, do a FAT32.

> And, also the repo you gave for bfdycrypt looks to be for TVOS... is that right?

no, dynastic repo it's for iOS.

**Reply by yodaluca23** [reply]
> > The only two options for File Systems are FAT32 and NTFS...
> > ![image](https://user-images.githubusercontent.com/67206487/201447661-051278c1-87bf-4516-b040-fa1670f17a5b.png)
> 
> Then, do a FAT32.
> 
> > And, also the repo you gave for bfdycrypt looks to be for TVOS... is that right?
> 
> no, dynastic repo it's for iOS.

Not the Sposify one, the one for bfdycrypt, [here ](https://www.ios-repo-updates.com/repository/diatrus-tvos-essentials/package/com.level3tjg.bfdecrypt/) that u gave above, it literally says "tvos-essentials" in the URL....

**Reply by RobyRew** [reply]
> Not the Sposify one, the one for bfdycrypt, [here ](https://www.ios-repo-updates.com/repository/diatrus-tvos-essentials/package/com.level3tjg.bfdecrypt/) that u gave above, it literally says "tvos-essentials" in the URL.... 

true, here's the one:
https://repo.p0358.net/

I just got that old iTVOS link from ios-repo-updates.com

**Reply by RobyRew** [reply]
> Not the Sposify one, the one for bfdycrypt, [here ](https://www.ios-repo-updates.com/repository/diatrus-tvos-essentials/package/com.level3tjg.bfdecrypt/) that u gave above, it literally says "tvos-essentials" in the URL.... 

true, here's the one:
https://repo.p0358.net/

I just got that old iTVOS link from ios-repo-updates.com

**Reply by yodaluca23** [reply]
@RobyRew ugggggg I can't install the Spotify version I need bc its only IOS 13 and up.... is there any Cydia tweak to spoof it or somehow let me install it anyways?

**Reply by RobyRew** [reply]
> @RobyRew ugggggg I can't install the Spotify version I need bc its only IOS 13 and up.... is there any Cydia tweak to spoof it or somehow let me install it anyways?

try installing this repo only tweak:

https://bisonsim.github.io

**Reply by yodaluca23** [reply]
> try installing this repo only tweak:
> 
> https://bisonsim.github.io

I installed it, restarted, rejailbroke, and it shows IOS 14.4 but none of my apps open except for Cydia, and Settings... not even the app store...

Also, do you mind if I credit you for helping me?

**Reply by RobyRew** [reply]
> I installed it, restarted, rejailbroke, and it shows IOS 14.4 but none of my apps open except for Cydia, and Settings... not even the app store...

Try using this other tweak, LowerInstall (I'm 99% sure it would work)
http://julioverne.github.io/description.html?id=com.julioverne.lowerinstall

If by any strange reason doesn't work, try this method (not recommended and **save your important data on the phone before** in case something bad happen.:
https://www.techacrobat.com/how-to-spoof-ios-version/

> Also, do you mind if I credit you for helping me?

If you want to do it, do it. I would not be against :)

**Comment by yodaluca23** [comment]
@RobyRew Did you get 8.7.46 to work at all? Because it isn't working for me... Some other cracks I've seen online use 8.6.42

Also, I can't get Bfdycrpt to work, I turn on the switch for Spotify, and it just shows the same app, with no decrypting GUI when opening it...

It works on all other apps I've tried, even the stock OS apps...

**Reply by yodaluca23** [reply]
Also, @RobyRew I have a iPhone 6s IOS 14.4 using CheckRa1n, but I recently heard of [Palera1n](https://github.com/palera1n/palera1n) that works on 15-16.2 on all Checkm8 devices iPhone 6s included, should I update to 15.7 and use [Palera1n](https://github.com/palera1n/palera1n) or just stick with 14.4 and Checkra1n. Cause it's also nice bc Palera1n has a windows version (installer..? Loader...? What is the software to jailbreak called..?) but idk...

**Comment by RobyRew** [comment]
just stick with checkra1n as is easier and maybe in the next future weeks is gonna be updated to give compatibility to iOS 15-16

**Comment by bachig26** [comment]
@yodaluca23 is there a crack/patch version of Spotify for windows/desktop similar to SpotilifeC and SpotC++.

**Reply by yodaluca23** [reply]
Yes! There is, check out [SpotX](https://github.com/amd64fox/SpotX) it's completely open source!

**Reply by bachig26** [reply]
thanks.

**Comment by bachig26** [comment]
~@yodaluca23 can you make a SpotCompiled Repo for both the apps to use for AltStore and SideStore?~

                          - sorry, forgot about the Readme 🙏
